ingestion/functions/common/common_lib.py

```diff
@@ -65,18 +66,21 @@ def finalize_upload(
         "summary": {"numCreated": count_created, "numUpdated": count_updated}}
     res = requests.put(put_api_url,
                        json=update,
-                       headers=headers)
+                       headers=headers,
+                       cookies=cookies)
     # TODO: Look for "errors" in res_json and handle them in some way.
+    # TODO: This can end up in a error-loop where we try to upload and fail
+    #       endlessly
```

Comment:
Thanks for getting this out of my :brain: and into the :computer:.

Reply:
nw, that scared me a bit when I noticed... might tackle that now actually.